content/en/developers/integrations/create-an-integration-detection-rule.md

```diff
@@ -0,0 +1,146 @@
+---
+further_reading:
+- link: https://docs.datadoghq.com/security/cloud_siem/log_detection_rules
+  tag: Documentation
+  text: Log Detection Rules
+kind: documentation
+title: Create an Integration Detection Rule
+description: Learn how to create a detection rule for your integration.
+---
+## Overview
+
+[Datadog Cloud SIEM (Security Information and Event Management)][1] unifies developer, operation, and security teams through one platform. Datadog provides a set of out-of-the-box detection rules for many features and integrations. View these rules in your [SIEM Detection Rules list][2].
+
+Create an out-of-the-box detection rule to help users find value in your Datadog integration. This guide provides steps for creating an integration-detection rule and best practices to follow during the creation process.
+
+To create a Datadog integration, see [Create a New Integration][3].
+
+## Create a detection rule
+### Build a detection rule JSON schema
+
+1. Navigate to **[Security > Cloud SIEM > Detection Rules > New Rule][4]** and create a new rule.
```

Comment:
@nimishaaa we will have to tell partner that currently only threshold rule are supported.